app/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: * 
+Disallow: /latest_version 
+Disallow: /latest_version.html 
+  
+Sitemap: https://kuma.io/sitemap.xml
```

Comment:
IIRC the robots.txt was pretty simple but there was a bunch a meta on the pages of non latest docs to skip indexing them.

Reply:
Will add this in a followup

Reply:
I've just investigated and the metadata is present on non-latest docs. They're set to `noindex,follow` which is correct